src/components/ConnectionHelp.js

```diff
@@ -0,0 +1,243 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  IconButton,
+  Dialog,
+  DialogContent,
+  Box,
+  Button,
+  Typography,
+  Link,
+} from '@material-ui/core';
+import {
+  withStyles,
+} from '@material-ui/core/styles';
+import { HelpOutline } from '@material-ui/icons';
+import { FormattedMessage } from 'react-intl';
+import { hot } from 'react-hot-loader';
+
+import RoverConnection from '@/containers/RoverConnection';
+import gigglebotImage from '@/assets/images/connection-help/gigglebot.png';
+import displayImage from '@/assets/images/connection-help/display.png';
+import flashingImage from '@/assets/images/connection-help/flashing.png';
+import deviceWindowsImage from '@/assets/images/connection-help/device-windows.jpg';
+
+// TODO: Use a flag on user model to launch modal immediately
+const mapStateToProps = ({ user }) => ({ user });
+
+class ConnectionHelp extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      host: window.location.host,
+      open: false,
+    };
+  }
+
+  handleOpen = () => {
+    this.setState({
+      open: true,
+    });
+  }
+
+  handleClose = () => {
+    this.setState({
+      open: false,
+    });
+  }
+
+  render() {
+    const { open, host } = this.state;
+    let env;
+    switch (host) {
```

Comment:
I think it would work better to make this a global variable passed in from the environment in `webpack.common.js`. The full URL can be put in that variable and this component can just use that like `SENTRY_DSN`

Reply:
Perfect. I'll make that change.